lib/foreman_maintain/cli.rb

```diff
@@ -33,6 +34,7 @@ class MainCommand < Base
       subcommand 'self-upgrade', 'Perform major version self upgrade', SelfUpgradeCommand
       subcommand 'maintenance-mode', 'Control maintenance-mode for application',
         MaintenanceModeCommand
+      subcommand 'report', 'Generate the usage reports', ReportCommand
```

Comment:
```suggestion
      subcommand 'report', 'Generate usage report', ReportCommand
```

I guess it's *one* report generated per run? :)

Reply:
right